### ISSUE TITLE
placement and styling of rob-bugson links stinks

### ISSUE THREAD

**Issue by willkg** [mozilla/rob-bugson#18]
I've been adding the rob-bugson links at the top of the PR page just below the heading. That means that you need to scroll up and down the page for certain use cases and that kind of stinks.

For example, you merge a PR and in order to click the link to add a merge comment to the bug, you have to scroll all the way back up the page.

I don't know what else we can do here, but it's make it more usable if we worked on this.

**Comment by willkg** [comment]
Maybe we should redo this to use the context menu? So then it would out the data on the fly and generates context menu links?

**Comment by biancadanforth** [comment]
Would https://github.com/willkg/socorro-release/issues/36#issuecomment-1771671562 help here? I.e. updating `rob-bugson` to match on `bug-*` rather than simply `bug *`, so we can leverage [GitHub autolinking](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#custom-autolinks-to-external-resources) which doesn't support spaces in the match patterns.

**Comment by willkg** [comment]
No, socorro-release#36 doesn't help here. This is more about how rob-bugson adds a set of links and currently does it in the space between the pr summary and the rest of the pr:

![image](https://github.com/willkg/rob-bugson/assets/820826/b541153e-f943-423e-bc5d-424289f05a40)

It's hard to get the text there and when GitHub changes the structure of the page, I have to go tinker. I was wondering if it'd be better as a context menu or something like that which is not part of the web page but rather part of the browser chrome.
